apps/rush-lib/src/logic/deploy/DeployArchiver.ts

```diff
@@ -59,7 +68,8 @@ export class DeployArchiver {
     // See: https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/tree/include/uapi/linux/stat.h#n10
     const permissionsValue: number = 0o120755;
 
-    const zip: JSZip = new JSZip();
+    // eslint-disable-next-line @typescript-eslint/typedef
```

Comment:
As I explained with `IYarnLockfileParseResult`, just use a regular `import` with a TODO comment saying to convert to  "import type" later.  *Don't remove our type annotations.*

Reply:
'JSZip' can only be imported by using 'import JSZip = require("../../../node_modules/jszip")' or by turning on the 'esModuleInterop' flag and using a default import.ts(2617)

Reply:
Can you just delay load DeployArchiver instead?

Reply:
Do you mean `importLazy('./DeployArchiver')` in `DeployManager.ts`?

Reply:
Yup - would that work?

Reply:
I am not sure. Let me try.

Reply:
This works:

```TypeScript
// TODO: Convert this to "import type" after we upgrade to TypeScript 3.8
import * as DeployArchiver from './DeployArchiver';
const da: typeof DeployArchiver = importLazy('./DeployArchiver');
```